homeassistant/components/axis/camera.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, device):
 
         MjpegCamera.__init__(
             self,
-            name=device.name,
+            name="",
```

Comment:
Hmmm should we adjust MjpegCamera for this? (Accepting optional name or `None`)?

Reply:
This is something akin to 4 year old code so it warrants a general revisit (Axis code)

Reply:
https://github.com/home-assistant/core/pull/75002

Reply:
Fixed, thanks!